packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -571,14 +642,34 @@ export const deleteSandbox: AsyncAction<string[]> = async (
   const { user } = state;
   if (!user) return;
   const oldSandboxes = state.dashboard.sandboxes;
-  actions.dashboard.deleteSandboxFromState(ids);
+  if (!isRepoPage) {
```

Comment:
Would it maybe be better that we also delete sandboxes from `state.dashboard.sandboxes.REPOS` in the `actions.dashboard.deleteSandboxFromState`? If you delete a sandbox tied to the GH repo from Drafts, it should also not be visible from Repositories I reckon.

Reply:
done